### PR TITLE
4 get api articles

### PR DESCRIPTION
**Pull request by majorhaji**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -28,6 +30,44 @@ describe("/api/topics", () => {
...
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles.length).toBe(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
```
Comment (B7Bprog) [inline __tests__/app.test.js]: Body should be excluded.

File: models/models.articles.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection.js");
+
+exports.selectArticles = () => {
+  const SQL = `SELECT articles.article_id, articles.author, articles.body, articles.created_at, title, topic, articles.votes, COUNT(comments.article_id)::int AS comment_count FROM articles LEFT JOIN comments ON comments.article_id =articles.article_id
```
Comment (B7Bprog) [inline models/models.articles.js]: Here as well, body should be excluded, please see Trello ticket No. 4.

**Review by B7Bprog — APPROVED** [review]
Please see my comments and amend your code to exclude the 'body' property, then good to merge. 👍

File: models/models.articles.js (hunk 2 above)
Comment (majorhaji) [reply models/models.articles.js]: ok thanks

**Pull request merged**